pkg/networkpath/traceroute/runner.go

```diff
@@ -249,3 +303,28 @@ func getHostname(ipAddr string) string {
 	}
 	return currHost
 }
+
+func createGatewayLookup() (network.GatewayLookup, uint32) {
```

Comment:
Should `createGatewayLookup` return an error when something is wrong and let caller handle/log the error? That seems more idiomatic in go.

Reply:
Yeah I think that makes sense, changed it to return errors and log them from the constructor instead